docs/windows.md

```diff
@@ -170,12 +170,12 @@ depending on whether you are using your own [signed](https://docs.microsoft.com/
 OVS kernel driver or you want to use the test-signed driver provided by Antrea,
 you will need to invoke the `Install-OVS.ps1` script differently (or not at all).
 
-| Containerized OVS daemons? | Test-signed OVS driver? | Run this command |
-| -------------------------- | ----------------------- | ---------------- |
-| Yes                        | Yes                     | `.\Install-OVS.ps1 -InstallUserspace $false` |
-| Yes                        | No                      | N/A |
-| No                         | Yes                     | `.\Install-OVS.ps1` |
-| No                         | No                      | `.\Install-OVS.ps1 -ImportCertificate $false -Local -LocalFile <PathToOVSPackage>` |
+| Containerized OVS daemons? | Test-signed OVS driver? | Run this command                                                          |
+| -------------------------- | ----------------------- |---------------------------------------------------------------------------|
+| Yes                        | Yes                     | `.\Install-OVS.ps1 -InstallUserspace $false`                              |
```

Comment:
Why is this still required?
- OVS is included in the container image
- The initContainer invokes `& $OVSInstallScript -InstallUserspace $false -LocalFile $mountPath\openvswitch`

So this seems redundant. IMO, after this change we have fully eliminated the need to run `Install-OVS.ps1` manually when using "containerized" OVS.

The only thing we haven't really resolved is for users bringing their own OVS as there is no simple way for them to include it in the image or mount it into the container.

Reply:
Do you mean we can delete the complete table from the guide or only keep "No Containerized OVS" cases?

Reply:
The table is less useful if the only case that requires running the script manually is the non-containerized (services) case. However, we can probably keep the table for now for clarity.
Assuming my understanding is correct and that for the Yes/Yes case we no longer need to run the script manually, then this table row should be updated with the "Run this command" column being set to "N/A".

Reply:
Got it, updated.